File: Curso python/exercicios/condicional/notas.py
n1 = float(input("Digite sua primeira nota: "))
n2 = float(input("Digite sua segunda nota: "))

nota_final = (n1 *2 + n2 * 2) / 2

print(f"NOTA FINAL = {nota_final}")

if nota_final < 60.00:
    print("REPROVADO")

File: curso C/notas.c
#include <stdio.h>

int main(int argc, char **argv)
{
	double nota1,nota2,notafinal;
	
	printf("Digite sua nota: ");
	scanf("%lf", &nota1);
	
	printf("Digite sua nota: ");
	scanf("%lf", &nota2);
	
	notafinal = nota1+nota2;
	
	printf("Sua nota final foi: %.1lf\n", notafinal);
	
	if (notafinal < 60){
		printf("REPROVADO");
	}
	
	return 0;
}

